### PR TITLE
Tests/accessibility identifiers

### PR DESCRIPTION
**Pull request by i336616**
Added a large number of accessibilityIdentifiers to UI elements in the project, to allow for UI testing.

## Checklist

* [x] Test your changes as thoroughly as possible before you commit them. Preferably, automate your test by unit/integration tests.
* [ ] If this PR comes from a fork, please [Allow edits from maintainers](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork)
* [x] Set a speaking title. Format: {task_name} (closes #{issue_number}). For example: Use logger (closes # 41)
* [ ] [Link your Pull Request to an issue](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) (if applicable)
* [ ] Create Work In Progress [WIP] pull requests only if you need clarification or an explicit review before you can continue your work item.
* [ ] Make sure that your PR is not introducing _unnecessary_ reformatting (e.g., introduced by on-save hooks in your IDE)

### CODE REVIEW

File: src/xcode/ENA/ENA.xcodeproj/xcshareddata/xcschemes/ENA.xcscheme [hunk 1]
```diff
@@ -37,7 +37,7 @@
       </BuildActionEntries>
    </BuildAction>
    <TestAction
-      buildConfiguration = "Debug"
+      buildConfiguration = "UITesting"
```
Comment (inf2381) [inline src/xcode/ENA/ENA.xcodeproj/xcshareddata/xcschemes/ENA.xcscheme:40]: Why UITesting in ENA? I thought we need it only for ENAUITests?

File: src/xcode/ENA/ENA.xcodeproj/xcshareddata/xcschemes/ENAUITests.xcscheme [hunk 2]
```diff
@@ -44,12 +44,20 @@
...
       selectedDebuggerIdentifier = "Xcode.DebuggerFoundation.Debugger.LLDB"
       selectedLauncherIdentifier = "Xcode.DebuggerFoundation.Launcher.LLDB"
+      language = "de"
       launchStyle = "0"
       useCustomWorkingDirectory = "NO"
       ignoresPersistentStateOnLaunch = "NO"
       debugDocumentVersioning = "YES"
       debugServiceExtension = "internal"
       allowLocationSimulation = "YES">
+      <EnvironmentVariables>
```
Comment (inf2381) [inline src/xcode/ENA/ENA.xcodeproj/xcshareddata/xcschemes/ENAUITests.xcscheme:54]: Do we need this environment variable?

Comment (i336616) [reply src/xcode/ENA/ENA.xcodeproj/xcshareddata/xcschemes/ENAUITests.xcscheme:54]: Looking at the CircleCI logs,

    xcodebuild -workspace ./ENA.xcworkspace -scheme ENAUITests

I don't think we do. Will delete, commit & push.

File: src/xcode/ENA/ENA.xcodeproj/xcshareddata/xcschemes/ENA.xcscheme (hunk 1 above)
Comment (i336616) [reply src/xcode/ENA/ENA.xcodeproj/xcshareddata/xcschemes/ENA.xcscheme:40]: Checking this locally, looks like we don't need it.

**Review by inf2381 — APPROVED** [review]

File: src/xcode/ENA/ENA/Source/Scenes/DynamicTableViewController/Views/DynamicTableViewImageCardCell.swift [hunk 3]
```diff
@@ -124,14 +124,16 @@ class DynamicTableViewImageCardCell: UITableViewCell {
 		cellImage.heightAnchor.constraint(equalToConstant: 130).isActive = true
 	}
```
Comment (EisvogelSAP) [inline src/xcode/ENA/ENA/Source/Scenes/DynamicTableViewController/Views/DynamicTableViewImageCardCell.swift:125]: I think that file might be gone. Changed need to be applied to `ExposureSubmissionImageCardCell`.

**Review by eisvogel — CHANGES_REQUESTED** [review]
`ExposureSubmissionImageCardCell needs to be updated, please.`

**Comment by i336616** [comment]
> `ExposureSubmissionImageCardCell needs to be updated, please.`

Updated now!

File: src/xcode/ENA/ENA/Source/Scenes/DynamicTableViewController/Views/DynamicTableViewImageCardCell.swift [hunk 4]
```diff
@@ -0,0 +1,159 @@
+// Corona-Warn-App
```
Comment (EisvogelSAP) [inline src/xcode/ENA/ENA/Source/Scenes/DynamicTableViewController/Views/DynamicTableViewImageCardCell.swift:1]: Nope, that file has to go.

**Review by eisvogel — CHANGES_REQUESTED** [review]
DynamicTableImageCardCell has to be removed, please.

**Review by eisvogel — APPROVED** [review]

**Pull request merged**